selenium_tests/cross_listing/page_objects/cross_listing_index_page.py

```diff
@@ -0,0 +1,12 @@
+from selenium.webdriver.common.by import By
+
+from selenium_tests.cross_listing.page_objects.cross_listing_base_page_object\
+    import CrossListingBasePageObject
+
+
+class Locators(object):
+    HEADING_ELEMENT = (By.XPATH, '//h3[contains(.,"Cross Listing")]')
```

Comment:
Great, it's conditioned against xpath right now because there is no ID element to check against the header.  I've requested for this id to be added when folks pick up TLT-1314.  Thanks for the feedback!   
https://jira.huit.harvard.edu/browse/TLT-1314?focusedCommentId=252942&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-252942